commitlintplugins.test.ts

```diff
@@ -219,3 +219,32 @@ test('type-space-after-comma2', () => {
     expect(typeSpaceAfterComma2.status).toBe(0);
 });
 
+test('wip-with-number-1', () => {
+    let commitMsgWithNoNumberAfterWIP = "WIP";
+    let wipWithNumber1 = runCommitLintOnMsg(commitMsgWithNoNumberAfterWIP);
+    expect(wipWithNumber1.status).not.toBe(0);
+});
+
+test('wip-with-number-2', () => {
+    let commitMsgWithNoNumberAfterWIP = "WIP: bla bla blah";
+    let wipWithNumber1 = runCommitLintOnMsg(commitMsgWithNoNumberAfterWIP);
+    expect(wipWithNumber1.status).not.toBe(0);
+});
+
+test('wip-with-number-3', () => {
+    let commitMsgWithNumberAfterWIP = "WIP01: bla bla blah";
+    let wipWithNumber1 = runCommitLintOnMsg(commitMsgWithNumberAfterWIP);
+    expect(wipWithNumber1.status).toBe(0);
+});
+
+test('wip-with-number-4', () => {
+    let commitMsgWithNumberAfterWIP = "wip0001111: bla bla blah";
+    let wipWithNumber1 = runCommitLintOnMsg(commitMsgWithNumberAfterWIP);
+    expect(wipWithNumber1.status).toBe(0);
+});
+
+test('wip-with-number-5', () => {
+    let commitMsgWithNumberAfterWIP = "wip: bla bla blah";
+    let wipWithNumber1 = runCommitLintOnMsg(commitMsgWithNumberAfterWIP);
+    expect(wipWithNumber1.status).not.toBe(0);
+});
```

Comment:
@parhamsaremi ok I've changed my mind slightly about this rule so do this now:

- First, delete tests 4 and 5.
- Change test 3 commit msg to just be "WIP1" (and `expect(wipWithNumber3.status).toBe(0)`)
- Change test 2 to be `expect(wipWithNumber3.status).toBe(0)`

Reply:
>  I assume you mean that a number or description is enough

Right.

> If so, I believe I should change the rule name and error too

Sure, and rule implementation to not need to use Regex, it can be super-simplified now. And while you're at it, please rebase the PR because it has conflicts now.

Reply:
I applied the required changes.